openfisca_us/variables/gov/local/ny/nyc/tax/income/ny_nyc_income_tax_before_credits.py

```diff
@@ -0,0 +1,49 @@
+from openfisca_us.model_api import *
+
+
+class ny_nyc_income_tax_before_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "NYC income tax before credits"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.NY
+
+    def formula(tax_unit, period, parameters):
+        # determine if tax unit is in NYC
+        county = tax_unit.household("county", period)
+        counties = county.possible_values
```

Comment:
To take a look at

Reply:
This is hitting #1342 too @PavelMakarchuk